lib/config.js

```diff
@@ -26,7 +26,8 @@ module.exports = class Config {
       throw new Error('[spike constructor] option "root" is required')
     }
     // merges API options into app.js options
-    let allOpts = Object.assign(this.parseAppJs(opts), opts)
+    let appJsOpts = this.parseAppJs(opts)
+    let allOpts = merge(appJsOpts, opts)
```

Comment:
There was when I was using the custom functions as it was a little easier to understand (and didn't mess about between contexts so much), but it may be better now to revert this line to have `this.parseAppJs(opts)` inline.